patroni/postgresql.py

```diff
@@ -292,6 +293,11 @@ def reload_config(self, config):
     def pending_restart(self):
         return self._pending_restart
 
+    @staticmethod
+    def configuration_allows_rewind(data):
+        return data.get('Current wal_log_hints setting', 'off') == 'on' \
+            or data.get('Data page checksum version', '0') != '0'
```

Comment:
Also, at the moment it checks only Postgres configuration. Is there any reason why doesn't it check Patroni configuration as well?